$NetBSD: patch-ac,v 1.2 2014/12/18 22:23:01 gdt Exp $

(Part of) patch to add option to not scan mail from authenticated
users, from:

  http://lists.freebsd.org/pipermail/freebsd-ports-bugs/2006-November/106024.html

--- spamass-milter.1.in.orig	2014-08-15 02:30:07.000000000 +0000
+++ spamass-milter.1.in
@@ -22,6 +22,7 @@
 .Op Fl r rejectmsg
 .Op Fl u Ar defaultuser
 .Op Fl x
+.Op Fl a
 .Op Fl S /path/to/sendmail
 .Op Fl - Ar spamc flags ...
 .Sh DESCRIPTION
@@ -218,6 +219,10 @@ Requires the
 flag.  The spamass-milter configuration process does its
 best to find sendmail, but it is possible to override this compiled-in
 setting via the
+.It Fl a
+Causes spamass-milter to pass through unchecked any messages from connections
+established using SMTP authentication.  This is useful for sites with
+remote usrs.
 .It Fl - Ar spamc flags ...
 Pass all remaining options to spamc. 
 This allows you to connect to a remote spamd with
